client/src/auth/ForgetPassword.js

```diff
@@ -0,0 +1,131 @@
+import {useState} from 'react';
+
+import { Link } from 'react-router-dom';
+
+import Spinner from '../utils/Spinner';
+import SuccessToast from '../utils/SuccessToast';
+
+// import axios from 'axios';
+
+const ForgetPassword = () => {
+    const [state,setState] = useState({email:"",otp:null,password:"",confirmPassword:""});
+    const [errors,setErrors] = useState({email:"",otp:"",password:"",confirmPassword:""});
+
+    const [isEmail,setIsEmail] = useState(false);
+    const [isOtp,setIsOtp] = useState(false);
+
+    const [spinner,setSpinner] = useState(false);
+    const [toast,setToast] = useState("");
+    const [showPassword,setShowPassword] = useState(false);
+
+    const handleChange = (e) => {
+       const {value,name} = e.target;
+       setState( state => ({...state,[name]:value}) )
+    }
+
+
+    const formValidate = (state) => {
+        let password,confirmPassword;
+ 
+        if(!/^(?=.*\d)(?=.*[a-z])(?=.*[a-zA-Z]).{8,}$/.test(state.password)){
```

Comment:
Create a variable for regex. That way, in future it'd be easy to make changes. Also, reusing a variable is easy, less chances of errors of using the regex itself.